src/env/schema.mjs

```diff
@@ -155,7 +156,8 @@ export const clientSchema = z.object({
   NEXT_PUBLIC_BASE_URL: z.string().optional(),
   NEXT_PUBLIC_UI_CATEGORY_VIEWS: zc.booleanString.default(true),
   NEXT_PUBLIC_UI_HOMEPAGE_IMAGES: zc.booleanString.default(true),
-  NEXT_PUBLIC_LOG_TRPC: zc.booleanString.default(false)
+  NEXT_PUBLIC_LOG_TRPC: zc.booleanString.default(false),
+  NEXT_PUBLIC_RECAPTCHA_KEY: z.string().optional()
```

Comment:
I think this should be required, otherwise the app will crash anyways at some point
```suggestion
  NEXT_PUBLIC_RECAPTCHA_KEY: z.string()
```